### PR TITLE
New multiselect

### PR DESCRIPTION
**Pull request by darcyadams**
screen shot 2018-08-14 at 3 48 41 pm
screen shot 2018-08-14 at 3 41 42 pm
screen shot 2018-08-14 at 3 41 49 pm

### CODE REVIEW

**Comment by darcyadams** [comment]
no tests yet, tests to follow, but want to get it available for QA

File: src/components/multi_select_option_item.coffee [hunk 1]
```diff
@@ -0,0 +1,46 @@
+React = require 'react'
+createClass = require 'create-react-class'
+PropTypes = require 'prop-types'
+
+{li, button, div} = require 'react-dom-factories'
+{ENTER} = require '../constants/keyboard'
```
Comment (meggantodd) [inline src/components/multi_select_option_item.coffee]: not seeing a button or a div

File: src/components/multi_select_option_list.coffee [hunk 2]
```diff
@@ -0,0 +1,152 @@
+React = require 'react'
+createClass = require 'create-react-class'
+PropTypes = require 'prop-types'
+Pvr = React.createFactory(require('./pvr'))
+map = require 'lodash/map'
+
+MultiSelectOptionItem = require './multi_select_option_item'
+
+{div, ul, li} = require 'react-dom-factories'
+
```
Comment (meggantodd) [inline src/components/multi_select_option_list.coffee]: seeing the ul only

File: src/components/multi_select_option_list.coffee [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      @listEl.scrollTop = focusedTop
+
+
+
+
+
+
+
+
+
```
Comment (meggantodd) [inline src/components/multi_select_option_list.coffee]: what's up with all this space

**Pull request merged**